Counterexamples/MapFloor.lean

```diff
@@ -59,50 +58,55 @@ local notation "ε" => (X : ℤ[ε])
 
 namespace IntWithEpsilon
 
-instance : LinearOrder ℤ[ε] :=
+instance nontrivial : Nontrivial IntWithEpsilon := inferInstance
+
+instance commRing : CommRing IntWithEpsilon := Polynomial.commRing
+
+instance : Inhabited IntWithEpsilon := ⟨0⟩
```

Comment:
Please add a porting note indicating that these used to be handled by a derive handler.